packages/services/f-braze-adapter/src/index.js

```diff
@@ -1,22 +1,4 @@
-import GetDispatcher from './BrazeDispatcher';
+import BrazeAdapter from './BrazeAdapter';
 
-/**
- * sessionTimeoutInSeconds
- * Set session timeout to 0 in order to avoid caching issues with Braze
- * @type {number}
- */
-export const sessionTimeoutInSeconds = 0;
+export default BrazeAdapter.initialise;
```

Comment:
Should we await here?  Is there a risk that we could return an as-yet uninitialised adapter?

Reply:
This exports the method, not the returned value.